pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/kits/KitDisplayNameTest.java

```diff
@@ -21,32 +21,30 @@ public class KitDisplayNameTest extends DbTxnBaseTest {
     private static String BLOOD_RNA_KIT_TYPE_DISPLAY_NAME = "BLOOD and RNA";
     static String[] subkitSettingsIds = null;
 
-    private static KitUtil
-            KitUtil = new KitUtil("test_instance", "test_instance_guid", "some_prefix", "test_group", BLOOD_RNA_KIT_TYPE_NAME);
+    private static TestKitUtil
+            testKitUtil = new TestKitUtil("test_instance", "test_instance_guid", "some_prefix", "test_group", BLOOD_RNA_KIT_TYPE_NAME);
 
     @BeforeClass
     public static void setupBefore() {
-        KitUtil.setupInstanceAndSettings();
+        testKitUtil.setupInstanceAndSettings();
     }
 
     @AfterClass
     public static void cleanUpAfterClass() {
-        KitUtil.deleteKitsArray();
+        testKitUtil.deleteKitsArray();
         if (subkitSettingsIds != null){
-            KitUtil.deleteSubKitSettings(subkitSettingsIds);
+            testKitUtil.deleteSubKitSettings(subkitSettingsIds);
         }
-        KitUtil.deleteInstanceAndSettings();
+        testKitUtil.deleteInstanceAndSettings();
 
     }
 
     @Test
-    public void testSubKitWithDisplayNAme(){
-        subkitSettingsIds = KitUtil.createSubKitsForTheStudy("BLOOD",  "BLOOD and RNA", 0, "RNA", null, 1);
-        Assert.assertEquals(2, subkitSettingsIds.length);
+    public void testKitWithDisplayNAme(){
```

Comment:
This test is working with the `createNonPepperTestKit` which is what is used for creating a juniper-likekit. Juniper Kit Creation is not set to use sub kits yet so instead I am just testing a single generic kit with a different display name and kit type name to make sure the kit returned has both values of kitTypeName and DisplayName